src/include/connection.h

```diff
@@ -312,6 +312,7 @@ struct __wt_connection_impl {
     uint32_t stat_flags; /* Options declared in flags.py */
 
     /* Connection statistics */
+    uint64_t rec_maximum_seconds; /* Maximum seconds reconciliation took. */
```

Comment:
This doesn't look quite complete to me - I expected to see a WT_STAT_SET and explicit reset call similar to the max page statistic:
```
$ ack evict_max_page_size src/
src/conn/conn_cache.c
300:    WT_STAT_SET(session, stats, cache_eviction_maximum_page_size, cache->evict_max_page_size);
src/evict/evict_page.c
205:    if (page->memory_footprint > conn->cache->evict_max_page_size)
206:        conn->cache->evict_max_page_size = page->memory_footprint;
src/include/cache.h
91:    uint64_t evict_max_page_size; /* Largest page seen at eviction */
src/txn/txn_ckpt.c
777:    conn->cache->evict_max_page_size = 0;
```

Reply:
Thanks, you're correct -- pushed in 92d61f92e.

I don't see any obvious clean way to do this (maybe it's time for a new class of "reconciliation" statistics, separate from the "cache" statistics?).

Let me know if this still doesn't look correct.